common/v2/features/PurchaseMembership/PurchaseMembershipStepper.tsx

```diff
@@ -21,38 +26,99 @@ const initialMembershipFlowState = {
 };
 
 const PurchaseMembershipStepper = () => {
-  const {
-    purchaseMembershipFlowState,
-    handleUserInputFormSubmit,
-    handleTxSigned
-  } = useStateReducer(MembershipInteractionFactory, initialMembershipFlowState);
+  const { purchaseMembershipFlowState, handleUserInputFormSubmit } = useStateReducer(
+    MembershipInteractionFactory,
+    initialMembershipFlowState
+  );
 
+  const { state, prepareTx, sendTx, stopYield, initWith } = useTxMulti();
+  const { canYield, isSubmitting, transactions } = state;
+  const { account, membershipSelected }: MembershipPurchaseState = purchaseMembershipFlowState;
+
+  // @ts-ignore Ignore type issue with signing step for now
```

Comment:
Is this something that could be fixed before this pr is merged?

Reply:
👍